test/integration/framework/framework.go

```diff
@@ -80,7 +80,7 @@ func BeforeSuite(ctx context.Context, crdPath string, mgrSetup ManagerSetup) (*r
 func AfterSuite(testEnv *envtest.Environment) {
 	ginkgo.By("tearing down the test environment")
 	err := testEnv.Stop()
-	gomega.ExpectWithOffset(1, err).NotTo(gomega.HaveOccurred())
+	gomega.ExpectWithOffset(1, err).To(gomega.Succeed())
```

Comment:
This was fine like it was.

It kind of reads: "expect error to not have occurred".

If you don't store the error in a variable, it reads: "expect [action] to succeed"

Reply:
ops, did this one by mistake, fixed.